Xamarin.PropertyEditing.Windows/EditorPropertySelector.cs

```diff
@@ -113,7 +114,11 @@ private bool TryGetTemplate (Type type, out DataTemplate template)
 			{ typeof(TypePropertyViewModel), typeof(TypeEditorControl) },
 			{ typeof(CollectionPropertyViewModel), typeof(CollectionEditor) },
 			{ typeof(RatioViewModel), typeof(RatioEditorControl) },
-			{ typeof(AutoResizingPropertyViewModel), typeof(AutoResizingMaskEditorControl) }
+			{ typeof(AutoResizingPropertyViewModel), typeof(AutoResizingMaskEditorControl) },
+			{ typeof(PropertyViewModel<char>), typeof(StringEditorControl) },
```

Comment:
These 2 are added to show "an Editor" and just using the Default Editors This may need fleshing out in future.

Reply:
allow editor providers would be awesome